Makefile

```diff
@@ -31,7 +31,6 @@ e2e: DOCKER_IMAGE?=$(shell ./scripts/fallback-docker.sh $(LOCAL_DOCKER_BASE):hea
 e2e: pkg/tests/testdata/traces-dataset.sz generate
 	go test -v ./pkg/tests/end_to_end_tests/ -timescale-docker-image=$(DOCKER_IMAGE)
 	go test -v ./pkg/tests/end_to_end_tests/ -use-timescaledb=false -timescale-docker-image=$(DOCKER_IMAGE)
-	go test -v ./pkg/tests/end_to_end_tests/ -use-multinode -timescale-docker-image=$(DOCKER_IMAGE)
```

Comment:
Could we comment this out instead, and add a note about why we've commented it out? See https://github.com/timescale/promscale/blob/e7745aa1b95ac4d58f18beef9d06924062f5ee4c/.github/workflows/go-e2e.yml#L83